exercises/gigasecond/instructions.md

```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+Your task is to determine the date and time one gigasecond after a certain date.
+
+A gigasecond is one thousand million seconds.
+That is a one with nine zeros after it.
```

Comment:
I didn't put `1,000,000,000` here, because number formatting is very much a thing that depends on localization. I know we've settled on "US English" as the standard, but if you come from a country that uses a comma instead of a decimal point, seeing the number with all the commas can feel weird and disorienting. I don't love using underscores, either. And spaces just looks weird.

I've not found a solid way of doing this that I'm happy with; any suggestions are welcome.

Reply:
US English [has a lot of number-related complexity](https://www.grammarbook.com/numbers/numbers.asp) around best practices.

I like "That is a one with nine zeros after it." It helps me understand/visualize what size number I might need to either convert or add into my date representation. And I agree - the _way_ a large number is written often confuses/slows me down - even commas, and _especially_ underscores.  

But if we consistently used underscores or spaces _everywhere_ maybe it would be less weird? 

Commas feel like a bad practice (_and there are different rules about year-numbers to add on top of everything else_).  I'd flat-out suggest underscores, but do worry slightly that that would lead some students to think they needed to format their input or output numbers like that.

Reply:
@BethanyG Thanks :)

> But if we consistently used underscores or spaces everywhere maybe it would be less weird?

We (katrina and I) tried underscores but it felt less approachable, more technical (even though it's not really).

I'm coming round to the idea that we use underscores but that we localise those when we render on the website. For now I think we go with just the text but I'll see if there's an easy way to achieve that localisation bump this week.